runtime/src/runtime/fvm.rs

```diff
@@ -385,6 +385,29 @@ where
         handle_send_result(to, method, fvm::send::send_read_only(to, method, params, None))
     }
 
+    fn send_with_gas(
```

Comment:
I think _most_ users will want a clean distinction between `send_read_only` and `send`. I know it's annoying, but I'd consider adding more functions (and implementing the non `with_gas` functions in terms of the main function.

Reply:
Dunno, I went for maximum comfort minimum churn here, but we can certainly revisit this.